pkg/kv/kvserver/kvflowcontrol/replica_rac2/processor.go

```diff
@@ -497,14 +441,10 @@ type processorImpl struct {
 		rc rac2.RangeController
 	}
 
-	// replMu contains the fields that must be accessed while holding Replica.mu.
-	replMu struct {
-		// raftNode provides access to a subset of raft RawNode. The reference is
-		// updated while holding both Replica.raftMu and Replica.mu, so can be read
-		// with any of the two mutexes locked. When interacting with raftNode, the
-		// Replica.mu must be held. See RaftNode comments.
-		raftNode RaftNode
-	}
+	// raftInterface is passed to RangeController. The reference is updated
+	// while holding both Replica.raftMu and Replica.mu, so can be read with any
+	// of the two mutexes locked.
```

Comment:
Btw, we have a new convention for this double mutexing since #131493. There is a `Replica.shMu` section now (which doesn't include all the things yet, but `RawNode` should be part of it). Consider having a section like this if the double mutexing is essential here.